hw/ip/otbn/doc/_index.md

```diff
@@ -61,49 +61,28 @@ The instruction set is split into two groups:
 
 ### General Purpose Registers (GPRs)
 
-OTBN has 32 General Purpose Registers (GPRs).
-Each GPR is 32b wide.
-General Purpose Registers in OTBN are mainly used for control flow.
-The GPRs are defined in line with RV32I.
+OTBN has 32 General Purpose Registers (GPRs), each of which is 32b wide.
+The GPRs are defined in line with RV32I and are mainly used for control flow.
 
-Note: GPRs and Wide Data Registers (WDRs) are separate register files.
-They are only accessible through their respective instruction subset:
-GPRs are accessible from the base instruction subset, and WDRs are accessible from the big number instruction subset (`BN` instructions).
```

Comment:
This section was added by explicit request during an earlier document review, IIRC.

Reply:
Sigh. Ok, I've sort of re-instated it: it's now part of the first paragraph but I've fixed it to be more accurate (since the big number instructions use GPRs for pointers and register indirection).

I assume that this sentence was supposed to be a "we're reasonably secure" notice, so I've instead written that the main data path uses a separate register file.